tests/phpunit/Integration/ByJsonScript/Fixtures/q-0702.json

```diff
@@ -117,6 +117,40 @@
 					}
 				]
 			}
+		},
+		{
+			"about": "#5 search phone number that does not contain 123",
+			"condition": "[[Has telephone number::!~*123*]]",
+			"printouts" : [],
+			"parameters" : {
+				"limit" : "10"
+			},
+			"queryresult": {
+				"count": "1",
+				"results": [
+					"Page/07/02/2#0##"
```

Comment:
Great, thank you for your insight. I have added the "wikidocu missing" label so I do not forget to do this on wiki for a wider audience to know this.